dotcom-rendering/src/model/guardian.ts

```diff
@@ -34,6 +34,7 @@ export interface Guardian {
 			isPaidContent?: boolean;
 			isDev?: boolean;
 			hasInlineMerchandise?: boolean;
+			userAttributesApiUrl?: string;
```

Comment:
Why has this feature been added?

Reply:
yes this was not explained, `userAttributesApiUrl` will be needed as part of set up in PR https://github.com/guardian/dotcom-rendering/pull/9362 , for the api call which was refactored:
`window.guardian.config.page.userAttributesApiUrl ??`